pkg/cloudprovider/aws/object_store.go

```diff
@@ -51,7 +51,7 @@ type objectStore struct {
 	signatureVersion string
 }
 
-func NewObjectStore(logger logrus.FieldLogger) cloudprovider.ObjectStore {
+func NewObjectStore(logger logrus.FieldLogger) velero.ObjectStore {
```

Comment:
export `objectStore` and its methods and return a pointer to it here instead of the interface

Reply:
Still need to do this.